packages/grafana-data/src/utils/arrayUtils.ts

```diff
@@ -12,6 +12,8 @@ export function moveItemImmutably<T>(arr: T[], from: number, to: number) {
  * Null/undefined/empty string values are always sorted to the end regardless of the sort order provided
  */
 const collator = new Intl.Collator(undefined, { numeric: true, sensitivity: 'base' });
+const defaultCompare = (a: number, b: number) => a - b;
```

Comment:
Sorry for not spotting this the first time, but maybe calling this `numericCompare` might be more accurate? This isn't actually the default comparator.